scripts/load-balance-aca-setup.sh

```diff
@@ -31,4 +39,4 @@ fi
 
 azd env set OPENAI_HOST azure_custom
 azd env set AZURE_OPENAI_CUSTOM_URL $fqdn
-
+echo "Set OPENAI_HOST to azure_custom and AZURE_OPENAI_CUSTOM_URL to $fqdn"
```

Comment:
If this is a script to help customers complete steps faster, then the echo line should indicate the set env variables are done. If you need the customer to manually complete these steps for some reason, then keep this line and remove the azd env set lines.

Reply:
This is the format the ACA infra outputs. 'https://${app.properties.configuration.ingress.fqdn}'

Reply:
Ah sorry this is meant to be "set" in the past tense, like "Script has set..." but that same as imperative in English! So I should reword.  I just want it to be clear what its done.